version_test.go

```diff
@@ -0,0 +1,14 @@
+package specs
+
+import (
+	"regexp"
+	"testing"
+)
+
+func TestVersion(t *testing.T) {
+	re := regexp.MustCompile(`\d+\.\d+\.\d+`)
```

Comment:
This _test.go can be dropped entirely. It's just a smoke test and not intended to be a SemVer complete assertion.